frontend/pages/settings/security.tsx

```diff
@@ -0,0 +1,53 @@
+import { useIntl } from 'react-intl';
+
+import { withLocalizedRequests } from 'hoc/locale';
+
+import { InferGetStaticPropsType } from 'next';
+
+import { loadI18nMessages } from 'helpers/i18n';
+
+import Settings2FA from 'containers/settings-security/2fa';
+import SettingsChangePassword from 'containers/settings-security/change-password';
+
+import Head from 'components/head';
+import LayoutContainer from 'components/layout-container';
+import { UserRoles } from 'enums';
+import NakedLayout from 'layouts/naked';
+import ProtectedPage from 'layouts/protected-page';
+import SettingsLayout, { SettingsLayoutProps } from 'layouts/settings';
+import { PageComponent } from 'types';
+
+export const getStaticProps = withLocalizedRequests(async ({ locale }) => {
+  return {
+    props: {
+      intlMessages: await loadI18nMessages({ locale }),
+    },
+  };
+});
+
+type InformationPageProps = InferGetStaticPropsType<typeof getStaticProps>;
+
+const Information: PageComponent<InformationPageProps, SettingsLayoutProps> = () => {
+  const { formatMessage } = useIntl();
+
+  return (
+    <ProtectedPage permissions={[UserRoles.ProjectDeveloper, UserRoles.Investor, UserRoles.Light]}>
+      <Head title={formatMessage({ defaultMessage: 'Change password', id: 'L4nXIc' })} />
```

Comment:
We probably want the page to be called “Account security” 😉. The other tab names the page “Account information”.